DI-Policies.md

```diff
@@ -0,0 +1,67 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
+      + Examples include, but are not limited to:
+         + Promotion of diversity-and-inclusion-minded vendors in my social feeds and blog
+         + Sharing love stories from underrepresented folx in my social feeds and blog
+
+   + Use my privilege, and speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website:
```

Comment:
@Pseudopam @bryanaknight Added specifics to audit stuff for both website and social. There's only two items for each, and I feel like there should be more I should be measuring, but I'm having a hard time identifying them right now. Maybe it'll be something that becomes more clear over the course of the year. Can you think of anything else I should be checking myself on regularly for either my website or social feeds?

Reply:
Maybe for the website add something like:
* Ensure D&I Policies are up-to-date and accurate based on ongoing work and evolving best practices